### PR TITLE
Allow for disabling the shutdown message

### PR DESCRIPTION
**Pull request by twisterghost**
By default, it is still on, but config.shutdown_message can toggle it off:

```crystal
# Disable the shutdown message
Kemal.config.shutdown_message false
```

### CODE REVIEW

File: src/kemal/config.cr [hunk 1]
```diff
@@ -42,6 +43,14 @@ module Kemal
       @logger = logger
     end
 
+    def shutdown_message(status : Bool)
```
Comment (sdogruyol) [inline src/kemal/config.cr]: `shutdown_message=`

File: src/kemal/config.cr [hunk 2]
```diff
@@ -42,6 +43,14 @@ module Kemal
...
+      @shutdown_message = status
+    end
+
+    def shutdown_message?
```
Comment (sdogruyol) [inline src/kemal/config.cr]: This can be implemented as a getter method like

```
getter :shutdown_message
```

File: src/kemal/config.cr [hunk 3]
```diff
@@ -17,6 +17,9 @@ module Kemal
     property host_binding, ssl, port, env, public_folder, logging, running,
       always_rescue, serve_static : (Bool | Hash(String, Bool)), server, extra_options
 
+    getter :shutdown_message
+    setter :shutdown_message
```
Comment (alex-lairan) [inline src/kemal/config.cr]: Why do not use `property` here? :)

Comment (twisterghost) [inline src/kemal/config.cr]: Thanks. Still learning crystal, should have looked up that line above 💃

**Comment by twisterghost** [comment]
This also fixes the other half of #290

**Comment by twisterghost** [comment]
Is there anything else you'd like me to update or is there a chance for this to be merged?

**Comment by sdogruyol** [comment]
Hey thanks for the PR. I wasn't sure about this till now 👍 

Could you please also send a PR documentation for this at https://github.com/kemalcr/kemalcr.com

**Comment by twisterghost** [comment]
Will do.

**Pull request merged**